docs/management/advanced-options.asciidoc

```diff
@@ -215,8 +215,9 @@ the changes.
 
 [horizontal]
 [[xpackdashboardmode-roles]]`xpackDashboardMode:roles`::
-**Deprecated. Use <<kibana-feature-privileges,feature privileges>> instead.**
-The roles that belong to <<xpack-dashboard-only-mode, dashboard only mode>>.
+deprecated:[7.7.0] The roles that belong to
```

Comment:
This tooltip does not display correctly because it follows a heading.

Reply:
Yes, I opened https://github.com/elastic/docs/issues/2096 and minimized the content in that tooltip to work around the issue for now.

Reply:
Since there doesn't seem likely to be a work around in the current docs system, I've reverted the changes in this file.